### ISSUE TITLE
ImageCacheIt element is being ignored.

### ISSUE THREAD

**Issue by Murilo-Perrone** [triniwiz/nativescript-image-cache-it#74]
### Make sure to check the demo app(s) for sample usage 
Yes.
First, installed the plugin: `tns plugin add nativescript-image-cache-it`
Then I copied everything needed from ng sample project to my project.
Then I tried many simple stuff like: `<ImageCacheIt src="~/images/ico2.png"></ImageCacheIt>` but no image was shown.
I also tested with a raw hello-world project :
`tns create my-app-name --template tns-template-hello-world-ng`
... then added all necessary code from imageCacheIt ng sample project. But all I see are the urls in text. the ImageCacheIt element is ignored.

### Make sure to check the existing issues in this repository 
This one has same symptom: https://github.com/triniwiz/nativescript-image-cache-it/issues/47

### If the demo apps cannot help and there is no issue for your problem, tell us about it
The ImageCacheIt xml element is being ignored.

### Which platform(s) does your issue occur on?
Android phone and Android emulator Pixel 2 XL API 28.

### Please, provide the following version numbers that your issue occurs with:
```
TNS 6.3.3
node_modules/tns-core-modules/package.json = "version": "6.3.2"
package.json
  "tns-android": { "version": "6.3.1" }
  "nativescript-image-cache-it": "^5.0.0-beta.14",
node v10.19.0
npm 6.13.4
Mac OS Mojave 10.14.3
```

### Please, tell us how to recreate the issue in as much detail as possible. 
Add an element to a XML page with a working src, and the element is ignored. No build or runtime are shown even with `--env.aot`.

### Is there any code involved? 
No specific code, but `TNSImageCacheItModule` is being imported in app.module.ts 
And also tried using `ImageCacheIt.enableAutoMM();` in main.ts

**Comment by Murilo-Perrone** [comment]
Oh my god, it worked. I realized the angular sample just needs to be updated. The `imageUri` attribute is **deprecated**.

Also, I had get around CLEARTEXT error from Glide component. I did i by adding `android:usesCleartextTraffic="true"` attribute in `App_Resources/Android/src/main/AndroidManifest.xml` on `<manifest ...><application ` element. Other solutions are available here:
https://stackoverflow.com/questions/45940861/android-8-cleartext-http-traffic-not-permitted

I'll try to update the main documentation and the angular sample project and do a pull request, then I will close this issue. Seems like gif isn't implemented for android ?
